nservicebus/azure-storage-persistence/configuration.md

```diff
@@ -54,4 +56,4 @@ For Timeouts:
 
 snippet:AzurePersistenceTimeoutsCustomization
 
-NOTE: Subscriptions and Timeouts persistence configuration only has an effect when used with Azure Storage Queues transport from NServiceBus Azure Version 6 and later.
\ No newline at end of file
+NOTE: Subscriptions and Timeouts persistence configuration only has an effect when used with Azure Storage Queues transport from NServiceBus.Azure Version 6.x and NServiceBus.Persistence.AzureStorage Version 1.x and above.
```

Comment:
Don't need the ".x"s

> Subscriptions and Timeouts persistence configuration only has an effect when used with...

What other contexts would you use these in? What mistake does this note prevent?